stages/org.osbuild.users

```diff
@@ -124,7 +131,7 @@ def usermod(root, name, gid=None, groups=None, description=None, home=None, shel
         subprocess.run(["chroot", root, "usermod", *arguments, name], check=True)
 
 
-def add_ssh_key(root, user, key):
+def add_ssh_key(root, user, keys):
```

Comment:
(super nitpick - sorry!) should this function name now also be plural, i.e. `add_ssh_keys()` ?

Reply:
Fixed.